Doc/Tutorial/chapter_entrez.tex

```diff
@@ -261,14 +262,13 @@ \section{ESummary: Retrieving summaries from primary IDs}
 \begin{verbatim}
 >>> from Bio import Entrez
 >>> Entrez.email = "A.N.Other@example.com"     # Always tell NCBI who you are
->>> handle = Entrez.esummary(db="journals", id="30367")
+>>> handle = Entrez.esummary(db="nlmcatalog", term="[journal]", id="101660833")
```

Comment:
What is ``term="[journal]"`` used for? It doesn't seem to have any useful effect comparing the two XML results, and looks like an erroneous addition to the URL to me.